### PR TITLE
Add descriptions to RDVertexAttribute's properties

### PR DESCRIPTION
**Pull request by DevPoodle**
This PR adds basic descriptions to all of RDVertexAttribute's properties.

### CODE REVIEW

File: doc/classes/RDVertexAttribute.xml [hunk 1]
```diff
@@ -10,14 +10,19 @@
 	</tutorials>
 	<members>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="RenderingDevice.DataFormat" default="218">
+			The way this attribute's data is interpreted when sent to a shader.
```
Comment (Mickeon) [inline doc/classes/RDVertexAttribute.xml:13]: Consistent with another description in this same PR
```suggestion
			The way that this attribute's data is interpreted when sent to a shader.
```

File: doc/classes/RDVertexAttribute.xml [hunk 2]
```diff
@@ -10,14 +10,19 @@
...
 		</member>
 		<member name="frequency" type="int" setter="set_frequency" getter="get_frequency" enum="RenderingDevice.VertexFrequency" default="0">
+			The rate at which this attribute is pulled from it's vertex buffer.
```
Comment (Mickeon) [inline doc/classes/RDVertexAttribute.xml:16]:
```suggestion
			The rate at which this attribute is pulled from its vertex buffer.
```

File: doc/classes/RDVertexAttribute.xml [hunk 3]
```diff
@@ -10,14 +10,19 @@
...
 		</member>
 		<member name="location" type="int" setter="set_location" getter="get_location" default="0">
+			The location in the shader that this attribute is bound to.
 		</member>
 		<member name="offset" type="int" setter="set_offset" getter="get_offset" default="0">
+			The amount of bytes between the start of the vertex buffer and the first instance of this attribute.
```
Comment (Mickeon) [inline doc/classes/RDVertexAttribute.xml:22]:
```suggestion
			The number of bytes between the start of the vertex buffer and the first instance of this attribute.
```

File: doc/classes/RDVertexAttribute.xml [hunk 4]
```diff
@@ -10,14 +10,19 @@
...
 		</member>
 		<member name="stride" type="int" setter="set_stride" getter="get_stride" default="0">
+			The amount of bytes between the starts of consecutive instances of this attribute.
```
Comment (Mickeon) [inline doc/classes/RDVertexAttribute.xml:25]:
```suggestion
			The number of bytes between the starts of consecutive instances of this attribute.
```

File: doc/classes/RDVertexAttribute.xml [hunk 5]
```diff
@@ -10,14 +10,19 @@
 	</tutorials>
 	<members>
 		<member name="format" type="int" setter="set_format" getter="get_format" enum="RenderingDevice.DataFormat" default="218">
+			The way this attribute's data is interpreted when sent to a shader.
 		</member>
 		<member name="frequency" type="int" setter="set_frequency" getter="get_frequency" enum="RenderingDevice.VertexFrequency" default="0">
+			The rate at which this attribute is pulled from it's vertex buffer.
 		</member>
 		<member name="location" type="int" setter="set_location" getter="get_location" default="0">
+			The location in the shader that this attribute is bound to.
```
Comment (Mickeon) [inline doc/classes/RDVertexAttribute.xml:19]: This property seems to be missing an unit of measure, if it needs one.

**Review by Mickeon — COMMENTED** [review]
Looks good, but I cannot say whether this is accurate information.

File: doc/classes/RDVertexAttribute.xml (hunk 5 above)
Comment (DevPoodle) [reply doc/classes/RDVertexAttribute.xml:19]: No, its more like a unique ID for each attribute than a specific location in memory.

**Review by Mickeon — APPROVED** [review]
I do approve, but I reiterate the rendering team should take a look at it to make sure.

**Review by clayjohn — APPROVED** [review]

**Comment by Repiteo** [comment]
Thanks!

**Pull request merged**